packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts

```diff
@@ -114,7 +114,7 @@ export async function createDeclarationCreator(
   // if the tsconfig _isn't_ in the package directory though, it's probably fine to memoize it
   // since it should just be a root level tsconfig
   let { service, options, program } = await (normalizePath(configFileName) ===
-  normalizePath(path.join(dirname, "tsconfig.json"))
+  path.join(dirname, "tsconfig.json")
```

Comment:
`path.join` already calls `path.normalize` under the hood so this was not quite needed

Reply:
path.normalize !== normalize-path. [normalize-path](https://www.npmjs.com/package/normalize-path) unixifies paths, path.normalize transforms paths into the platform path separator

Reply:
Well, sure - `path.normalize` for example preserves a trailing slash **but** this quite can't if we are dealing with a `path.join` results. I honestly can't name what difference does it make here - maybe there is something obscure that I'm forgetting but I've done some tests and looked into `path.normalize` & `path.join` implementations and I think this is safe to be done

Reply:
normalize-path always returns paths with `/`, path.normalize will return paths with `\` on Windows so with the change on this line, the two will never be equal on Windows

Reply:
Ah, right - I was looking into the code that was executing on my machine but I've missed that the `path` module switches the whole implementations: 
https://github.com/nodejs/node/blob/fe43bd8ddd16a614ba157c57c438cd2f206c5603/lib/path.js#L1420